docs: simplify the react quick start (#1652)
alchemyplatform/aa-sdk: docs/pages/react/quickstart.mdx

---
title: React Quickstart
description: Learn how to get started with Alchemy Smart Wallets in React.
text: Send user operations
link: /react/send-user-operations
slug: wallets/react/quickstart
---

<div style={{ display: "flex", flexWrap: "wrap", alignItems: "flex-start" }}>
  <div style={{ flex: "1", marginRight: "10px" }}>

>     Get started with Alchemy Smart Wallets by choosing the path that best fits

Reviewer: ```suggestion
```

>     your project. Whether you're starting fresh or integrating into an existing

Reviewer: ```suggestion
Whether you're starting fresh or integrating into an existing
```

>     app, we'll guide you through creating embedded wallets with email, passkey,

Reviewer: ```suggestion
    app, we'll guide you through creating embedded wallets with social login and sending sponsored transactions from the smart wallets.
```

>     and social login for smart accounts.

Reviewer: ```suggestion
```

  </div>
</div>

<Tip>
  Check out what you can build using our [Demo App](https://demo.alchemy.com/).
</Tip>

## Choose your path:

<CardGroup cols={2}>
  <Card

>     title="Quickstart (New Project)"

Reviewer: ```suggestion
    title="Quickstart"
```

    href="/wallets/react/quickstart/new-project"
    icon="forward"
  >
    <p>**Starting from scratch?**</p>
    <p>**Time:** 5 minutes</p>
    <p>**What you need:** Node.js + Alchemy account</p>
    <p>
      **You'll get:** Complete new NextJS app with embedded wallets, social

>       login, and gas sponsorship pre-configured

Reviewer: ```suggestion
      login, EOA wallet connection, and gasless transactions.
```

    </p>
  </Card>
  <Card
    title="Add to Existing Project"
    href="/wallets/react/quickstart/existing-project"
    icon="bullseye-pointer"
  >
    <p>**Have an existing React app?**</p>
    <p>**Time:** 15-20 minutes</p>
    <p>**What you need:** Existing React/NextJS project + Alchemy account</p>
    <p>

>       **You'll get:** Embedded wallets added to your current app with custom

Reviewer: ```suggestion
      **You'll get:** Embedded wallets added to your current app with social login, batching transactions, and gas sponsorship.
```

>       authentication setup

Reviewer: ```suggestion
```

    </p>
  </Card>
</CardGroup>

## Not sure which to choose?

> **Choose Quickstart (New Project) if:**

Reviewer: ```suggestion
**Choose Quickstart if:**
```

- You're building a new project from scratch
- You want the fastest way to see embedded wallets in action (5 minutes!)

> - You're okay with a pre-built setup and want to customize later

Reviewer: ```suggestion
```
